docs/setup/provision.md

```diff
@@ -0,0 +1,12 @@
+# Connect Create® 3 to Wi-Fi
+## The Basics
+Follow the main guide for getting started [here](https://edu.irobot.com/create3-setup).
+
+## Select RMW Implementation
+![Application Configuration Detail](data/appconfig.png)
+
+If you are planning to use ROS 2, make sure you have selected the correct RMW implementation.
```

Comment:
clarify "correct": both machines using ROS should have the same implementation set.

Reply:
Clarified in 76164cbd58a389797cf0dc1f3fe42fe21a7d178c